website_sale_qty/README.rst

```diff
@@ -0,0 +1,74 @@
+.. image:: https://img.shields.io/badge/license-LGPL--3-blue.svg
+   :target: https://www.gnu.org/licenses/lgpl.html
+   :alt: License: LGPL-3
+
+==========================
+Website Sale - Price Tiers
+==========================
+
+This module looks at the price rules for a product and uses them to calculate 
+price tiers (quantity ranges with different unit pricing). It then adds radio 
+buttons to product pages in the website shop, which allow users to easily 
+select the minimum quantity corresponding to each tier.
+
+Installation
```

Comment:
Sure thing. I'll drop them. I figured I needed to put something there to align with the OCA template.